extensions/src/main/java/org/pentaho/platform/plugin/action/ActionInvokerSystemListener.java

```diff
@@ -48,73 +49,96 @@
   private static final String WORK_ITEM_FILE_EXTENSION = ".json";
   private static final String DEFAULT_CONTENT_FOLDER = "system/default-content";
   private String environmentVariablesFolder;
-
+  private String ERROR = "error";
+  private String OK = "ok";
+  private String WI_STATUS = "/wi-status.";
```

Comment:
ok, didn't know about this, thanks for pointing it out! Will be useful in the future